### DISCUSSION TITLE
不支持谷歌云多团队盘

### DISCUSSION THREAD

**Discussion by yzg9540** [AlistGo/alist#286] [Ideas]
如果所有的团队盘都想显示出来，或者展示多个
怎么弄呢，求大佬更新

最好是挨个id添加，或者获取后可以显示展示哪些？

**Comment by xhofe** [comment]
支持团队盘啊 根目录file_id填团队盘的就行了

**Reply by yzg9540** [reply]
> 支持团队盘啊根目录file_id 填团队盘的就行了

老哥，我的意思是，临时来看程序只支持手动填写的一个id
例如我有10个团队盘，怎么都展示出来，或者选择可以展示出来哪几个？

**Reply by xhofe** [reply]
添加十次

**Reply by yzg9540** [reply]
> 添加十次

懂了，感谢老哥。我本来还想在一个界面操作来，没想到还可以这样啊

**Reply by Minerchu** [reply]
一个一个添加快捷方式到自己的云盘下一个文件夹，然后挂载自己云盘里的这个文件夹，就等于全部挂载，（夸我
